src/bootstrap-table.js

```diff
@@ -829,6 +830,10 @@
                     that.header.stateField = column.field;
                     that.options.singleSelect = true;
                 }
+                
+                if(column.escape) {
+                	that.options.escapeColumn = true;
```

Comment:
Woops, good catch that was from earlier while I was testing it out.  I will remove it.